framework/math/quadratures/angular/sldfe_sq_quadrature.cc

```diff
@@ -787,181 +743,84 @@ SimplifiedLDFESQ::Quadrature::TestIntegration(int test_case, double ref_solution
   char buff0[200], buff1[200], buff2[200];
   snprintf(buff0, 200, "Riemann integral: %.20e\n", I_riemann);
   snprintf(buff1, 200, "Quadrature integral: %.10e\n", I_quadrature);
-  snprintf(buff2,
-           200,
-           "Error_RQ%05d_%06d: %2d %f %e\n",
-           Nd,
-           Nd * 8,
-           initial_level_,
-           h,
+  snprintf(buff2, 200, "Error_RQ%05d_%06d: %2d %f %e\n", Nd, Nd * 8,
+           initial_level_, h,
            std::fabs((I_riemann - I_quadrature) / ref_solution));
 
   log.Log() << buff0;
   log.Log() << buff1;
   log.Log() << buff2;
 }
 
-void
-SimplifiedLDFESQ::Quadrature::PrintQuadratureToFile(const std::string& file_base)
-{
+void SimplifiedLDFESQ::Quadrature::PrintQuadratureToFile(
+    const std::string &file_base) {
   log.Log() << "Printing SLDFE-Quadrature to file.";
 
-  std::ofstream vert_file, cell_file, points_file, python_file;
-  vert_file.open(file_base + "verts.txt");
+  std::ofstream vert_file, cell_file, points_file;
+
+  // Vertex file generation for plotting - is written to the path above
+  vert_file.open(file_base + "verts.csv");
```

Comment:
i would add `_` in front of points, so that the users do not need to provide a basename that includes `_`